core-plugins/src/test/java/io/cdap/plugin/transform/ProjectionTransformTest.java

```diff
@@ -104,31 +115,66 @@ public void testConfigurePipelineSchemaValidation() throws Exception {
     Assert.assertNull(mockConfigurer.getOutputSchema());
   }
 
-  @Test(expected = IllegalArgumentException.class)
+  @Test
   public void testSameFieldMultipleConverts() throws Exception {
     ProjectionTransform.ProjectionTransformConfig config = new ProjectionTransform
       .ProjectionTransformConfig(null, null, "x:int,x:long", null);
     Transform<StructuredRecord, StructuredRecord> transform = new ProjectionTransform(config);
     TransformContext transformContext = new MockTransformContext();
-    transform.initialize(transformContext);
+    String caughtException = "";
+    try {
+      transform.initialize(transformContext);
+    } catch (Exception e) {
```

Comment:
this should catch ValidationException and get the ValidationFailures from the exception.

Reply:
same with rest of the test cases

Reply:
Fixed